packages/migration_tool/controller.php

```diff
@@ -51,11 +51,17 @@ class Controller extends Package
     protected $singlePagesToExclude = array(
     );
 
-    protected $singlePageTitles = array(
-        '/dashboard/system/migration' => 'Migration Tool',
-        '/dashboard/system/migration/import' => 'Import Content',
-        '/dashboard/system/migration/export' => 'Export Content',
-    );
+    protected function getSinglePageTitle($path)
+    {
+        switch ($path) {
+            case '/dashboard/system/migration':
+                return t('Migration Tool');
```

Comment:
We need to wrap page names in `t` so that these strings can be automatically extracted from `c5:package-translate`